### PR TITLE
explain astro.conf in README.md

### PR DESCRIPTION
**Pull request by guzzisti**
Not sure if you are interested in PRs at all. I'll send that along anyway. :)

When playing around with astro i realized it supports customization via a config file, but this wasn't mentioned anywhere.
I added a few words in the README.md explaining the config format.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,25 @@ no arguments takes you to *gemini.circumlunar.space*:
...
 
+### Configuration
+
+You can setup a config file at `~/.config/astro/astro.conf` to configure *astro* the way you like.
+
+The file uses a simple `key=value` style, see the complete example for the default values below. The `style-` keys must be ANSI style codes.
+
+```
+margin=8
+homepage=gmn.clttr.info
```
Comment (blmayer) [inline README.md:53]:
```suggestion
homepage=gemini.circumlunar.space
```

**Review by blmayer — CHANGES_REQUESTED** [review]

**Comment by blmayer** [comment]
Hi guzzisti, thanks for taking your time, and yes, contributions are welcome.

That is a nice catch, I didn't notice this missing part, but just one small change before I accept your PR:

Please change the homepage setting to the default value I've been using (*gemini.circumlunar.space*), since I think is a more "neutral" website. I added the suggestion to the commit for your your convenience.

And thanks again!

**Comment by guzzisti** [comment]
lol, sorry, this wasn't meant to be there

**Review by blmayer — APPROVED** [review]

**Comment by blmayer** [comment]
It's merged! Thank you for contributing.

**Pull request merged**